browser/profiles/brave_profile_manager.cc

```diff
@@ -120,6 +121,7 @@ void BraveProfileManager::DoFinalInitForServices(Profile* profile,
     return;
   brave_ads::AdsServiceFactory::GetForProfile(profile);
   brave_rewards::RewardsServiceFactory::GetForProfile(profile);
+  brave_wallet::BraveWalletServiceFactory::GetServiceForContext(profile);
```

Comment:
Will start wallet service at the start of the browser session to keep P3A values up to date

Reply:
Thanks for the comment. It's good to put something like this in the body of the commit message, so the motivation of a change is easier to track down when reading the code later.